### PR TITLE
[FIX] fix issue:Module product_autocompany breaks travis ci test #10

### PR DESCRIPTION
**Pull request by duanyp1991**
by adding a migration script.
 Changes to be committed:
    new file:   product_autocompany/migrations/7.0.0.0/post-migration.py

### CODE REVIEW

File: product_autocompany/migrations/7.0.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+##############################################################################
+
+
+def migrate(cr, version):
+    if not version:
+        # this is run only when the module is first installed
+        # fill in the default company for products whose company is empty
+        cr.execute("update product_template"
+                   "SET company_id="
+                   "   (SELECT id from res_company limit 1);")
```
Comment (pedrobaeza) [inline product_autocompany/migrations/7.0.0.0/post-migration.py]: `WHERE company_id is NULL`

**Comment by duanyp1991** [comment]
Hi @pedrobaeza  I added a migration script for this module, but I have a problem that: it's hard to determine which company to assign to product as default.

This module doesn't work well for multi-company environment if one product is shared among different companies. I think for product, the idea should be the same as my module: partner_multi_company: adding a new field: company_ids.

**Comment by pedrobaeza** [comment]
Yeah, I know, but at least this script provides technical (but not business) correctness. Take into account that the cases where company_id is not set are few.

And indeed, ideally we should tend to a module like _partner_multi_company_.

**Comment by yvaucher** [comment]
I think this fix is ok to keep it simple and stupid. When installing this module you should be aware that you cannot share your products anymore.

So :+1:

**Pull request merged**